app/styles/upf-utils.less

```diff
@@ -1,3 +1,4 @@
+@import 'node_modules/@upfluence/oss-components/app/styles/oss-components';
```

Comment:
Why are you doing this import?

Reply:
Oss components styles are not loaded unless I add it here.
Was guided by @phndiaye 😄  (sous le bus 🚌 )